pages/03-core/solidgauge.page.tsx

```diff
@@ -17,6 +17,19 @@ const series: Highcharts.SeriesOptionsType[] = [
     name: "CPU Usage",
     type: "solidgauge",
     data: [randomInt(60, 90)],
+    showInLegend: true,
+  },
+  {
+    name: "Memory Usage",
+    type: "solidgauge",
+    data: [randomInt(40, 80)],
+    showInLegend: true,
+  },
+  {
+    name: "Disk Usage",
+    type: "solidgauge",
+    data: [randomInt(20, 70)],
+    showInLegend: true,
```

Comment:
nit: In the current demo the solidgauge colors are not well defined for the dark mode. It would be nice to update the test page accordingly. If necessary, more design tokens can be set in the core chart to support dark mode.

<img width="817" height="532" alt="Image" src="https://github.com/user-attachments/assets/968a3287-59e5-43e5-abc5-8344536ab7ca" />

Reply:
Sometimes after hot-reloading the gauge colours change at random (I did not set these). This might indicate an issue in the implementation.

<img width="817" height="532" alt="Image" src="https://github.com/user-attachments/assets/d34a3ae5-2f6a-45c7-bbbe-3171c2f9849f" />

Reply:
Lastly, doing the `{...omit(chartProps.cartesian, "ref")}` is not very accurate here either - I would revisit that. I think it is enough to pass a few needed props, but not destructure them all.

In the chartProps.cartesian we define the verticalAxisTitlePlacement="top" as default. When it is passed to this chart, the Y-axis title is shown at the top, which does not look expected. By default, the Y-axis title is not shown at all it seems.

Reply:
One more note: unlike in pie chart, there are no borders between segments in the gauge. That is likely a major a11y issue, too, as the users that are unable to differentiate certain colors might be unable to see where one segment ends, and the second one starts.